api/swagger.yml

```diff
@@ -468,6 +468,22 @@ components:
           type: boolean
           default: false
 
+    CommitOverrides:
+      type: object
+      properties:
+        message:
+          type: string
+          description: replace the commit message
+        metadata:
+          type: object
+          description: update the metadata of the commit record. New keys will be added to the original metadata, and existing keys will be overwritten.
+          additionalProperties:
+            type: string
+        date:
```

Comment:
I wonder if we really need this. Overriding dates is very uncommon, doing it in a revert or a cherry-pick? Seems very unlikely. I think we should start without this field and add it at a later stage if we need to.

Reply:
In the original issue, you mentioned "commit timestamp" - is that not it?

Reply:
It is, but it was just grocery list of commit properties without a strong product justification. While I can think of reasons to pass a new commit message and metadata, for the commit time we should just use `time.Now()` and not the original commit timestamp or one given by the user.